src/options/views/script-item.vue

```diff
@@ -281,6 +285,8 @@ $removedItemHeight: calc(
 
 .script {
   position: relative;
+  display: grid;
+  grid-template-columns: $iconSize 1fr auto;
```

Comment:
It requires Chrome 57, so minimum_chrome_version and browserlist should be changed accordingly.

Reply:
Ah yes, I'll do that since users are less likely to stick to an old version of Chrome.

Reply:
BTW, last time I checked AMO statistics, FF 52 was still used by a few hundred of users, probably because it's the last version for Windows XP. As for Chrome, its last XP-compatible version is 49, so we can raise our minimum to something more sensible, for example 86 used by the current release of Centbrowser.

Reply:
Chrome 80 implements a [lot of good stuff](https://caniuse.com/?compare=chrome+55,chrome+80&compareCats=all#:~:text=prefers%2Dcolor%2Dscheme) natively (optional chaining, spreading in objects, flat, flatMap, SVG favicon, `prefers-color-scheme`, `#rrggbbaa` to name a few), so that would make sense as a minimum if we bump FF minimum to 68, which was a ESR in 2019-2020.

Reply:
I tested on Firefox 52 and it worked well. I think we'd better keep supporting Firefox 52 to make things simple (without the need to create a separate version for FF52).
As for the new features, most of them are supported through Babel and we don't feel it so I guess we can keep it this way.

Reply:
What about `minimum_chrome_version` in manifest.json? It's necessary to forbid installing the extension in old Chrome otherwise it would show the script list incorrectly.

Reply:
1. Hmm, just as I feared `grid` is slower than `flex`. The first frame is now shown in ~380ms on average whereas previously it was in ~345ms. There's always now a blank paint frame when opening the dashboard (or reloading this tab). It's too much of a regression for such an insignificant convenience for us when writing CSS, so unless this can be fixed I'm against grid.
2. The edit button `</>` is now shown in table view mode, but it shouldn't be.
3. The script names in the table view mode are blue now, should be standard color.

Reply:
> There's always now a blank paint frame

I don't really see this, even without `batchRender.limit` I got everything out in the first paint frame. (~80 scripts)
![image](https://user-images.githubusercontent.com/3139113/200172844-cf182ae3-72d4-4fc5-bcd9-942abc3b13dc.png)

I think it's totally OK for the dashboard to have such a delay if there is no flashing.

Reply:
There's no empty frame anymore but the performance is still considerably worsened (the time to paint the first meaning frame) from ~345ms to ~380ms.

Reply:
Re-tested in my main browser with less scripts (60) than in Canary (135) and the first paint frame time is the same as `master`, no regression.